model-optimizer/extensions/front/MatMul_normalizer.py

```diff
@@ -90,6 +91,10 @@ def replace_sub_graph(self, graph: Graph, match: [dict, SubgraphMatch]):
         node.out_port(0).get_connection().set_source(bias_node.out_port(0))
```

Comment:
could you please re-write this transformation a bit to use `find_and_replace_pattern(self, graph: Graph)` and traverse all `Gemm` nodes by `for` loop? We do so for all the single node pattern